torch_xla/csrc/reduction.cpp

```diff
@@ -355,19 +355,25 @@ xla::XlaOp BuildMaxInDims(xla::XlaOp input,
 
 xla::XlaOp BuildMinInDim(xla::XlaOp input, xla::int64 dim,
                          bool keep_reduced_dimensions) {
+  return BuildMaxInDims(input, {dim}, keep_reduced_dimensions);
```

Comment:
I think you meant `BuildMinInDims` here.

Reply:
Thanks for catching this. Done.